helm/secdash/dags/mark-provisioned.py

```diff
@@ -0,0 +1,85 @@
+import os
+import requests
+from pymongo import MongoClient
+
+
+def fetch_products_mark_completed(provisioner_api_url, mark_provisioned_url, mongo_connection_url, env_id):
+    """
+    Fetches Approved Requests from MongoDB, makes call to the Provisioner by every of them
+    and if labels completed = true and phase = Succeeded, makes a call to callback URL.
+
+    Parameters:
+    - provisioner_api_url (str): The url for provisioner API that returns the workflow status
+    - mark_provisioned_url (str): The url to Registry App API that marks Product as Completed
+    - mongo_connection_url (str): The connection string to Registry App DB to fetch products in 'Approved' state
+    - env_id (str): The name of the environment (prod or test) to filter out non-lab clusters in test
+
+    Raises:
+    - Any exceptions that occur during the execution.
+    """
+
+    try:
+        # Establish MongoDB connection
+        client = MongoClient(mongo_connection_url)
+        database = client["pltsvc"]
+        requests_collection = database["PrivateCloudRequest"]
+        requested_projects_collection = database["PrivateCloudRequestedProject"]
+
+        product_requests = requests_collection.find({"decisionStatus": "APPROVED"}, projection={
+                                                    "_id": True, "licencePlate": True})
+
+        for request in product_requests:
+            request_id = request["_id"]
+            licence_plate = request["licencePlate"]
+            # a call to the RequestedProject table to get cluster
+            requested_project_document = requested_projects_collection.find_one(
```

Comment:
let's use `request["decisionDataId"]` to fetch the `PrivateCloudRequestedProject` document.